src/style.css

```diff
@@ -1 +1,44 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 1%;
+}
+
+.logo {
+  margin-left: 2.5%;
+  flex-grow: 3;
+}
```

Comment:
1. you wont require both   `margin-left` and `flex-grow` after you are done with suggestion for `header` class
2. add height and width desired for the logo here